hack/demo.sh

```diff
@@ -48,10 +48,11 @@ spec:
   configuration:
     exclude:
     - not_useful
```

Comment:
Repeating Zoran's suggestion, the `not_useful` doesn't exist any more, so no need to exclude it. Not sure about conftest-clair. I remember it used to regularly fail due to a time out. Does it pass more often now?

Reply:
I didn't see conftest-clair fails now, to see the latest ec validation result, run [simple-demo.sh](https://github.com/hacbs-contract/ec-cli/blob/main/hack/simple-demo.sh) which is with [quay.io/redhat-appstudio/ec-golden-image:latest](https://quay.io/repository/redhat-appstudio/ec-golden-image?tab=tags&tag=latest])

Reply:
I think conftest-clair was problematic in the past, timeouts against the API and such. IIRC the test stream now runs with a standalone clair tool, so that improved the reliability.